src/components/select_team_view.js

```diff
@@ -28,15 +28,15 @@ export default class SelectTeamView extends Component {
     }
 
     componentWillReceiveProps(nextProps) {
-        const {currentTeamId} = nextProps.teams;
+        const {currentTeamId} = nextProps.team;
```

Comment:
I know it's just changing plurals, but I noticed `componentWillReceiveProps` seems funky.